modules/ocm-networking-tab-adding-ingress.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// ocm/ocm-overview.adoc
+:_content-type: PROCEDURE
+[id="ocm-networking-tab-adding-ingress_{context}"]
+= Adding a network ingress to your {product-title} cluster
+
+You can add a network ingress to your cluster from the {cluster-manager-url} web UI.
+
+.Prerequisites
+
+* You have a Red Hat account.
+* You have the required permissions to make changes to your cluster in {cluster-manager}.
+
+.Procedure
+
+. From the **Networking** tab in {cluster-manager}, click the "Additional application router" toggle to enable the ingress. There are two options you can add to the additional router:
+.. **Make router private**: This checkbox allows you to control cluster privacy. By default, your ingress router is publicly exposed allowing anyone access. If you intend to run an application or website with access limited to individuals within your own organization, such as employees, then you can click this checkbox to make the network ingress private and requiring a private connection, such as a virtual private network (VPN) or virtual private cloud (VPC) peering connection to access. 
```

Comment:
The second sentence is really long and might need split into two. That might be difficult to work through since it's all one idea.